hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionConfiguration.java

```diff
@@ -76,6 +76,11 @@ public class ConnectionConfiguration {
   public static final String HBASE_CLIENT_META_SCANNER_TIMEOUT =
     "hbase.client.meta.scanner.timeout.period";
 
+  public static final String HBASE_CLIENT_USE_SCANNER_TIMEOUT_FOR_NEXT_CALLS =
+    "hbase.client.use.scanner.timeout.for.next.calls";
```

Comment:
Our scanner timeout configuration called `hbase.client.scanner.timeout.period`, so maybe this one should be `hbase.client.use.scanner.timeout.period.for.next.calls`. I dunno. Maybe it's clear enough as it is? What do you think?

Whatever its called, please add a comment about this new flag in https://hbase.apache.org/book.html#config_timeouts

Reply:
Further troubling, configuration points defined here are not populated into the online book automatically. I think you need to add the config, a default, and a description to hbase-defaults.xml in order for it to automatically gain mention in the book.

Reply:
I changed the property name as requested. I added https://issues.apache.org/jira/browse/HBASE-28098 to update book.

I'm not sure we should add this to hbase-default.xml. It's more of a feature flag, and I think of hbase-default.xml as holding defaults for common configs. I dunno though, we are very inconsistent there and im not sure of the criteria for adding (many many many configs are missing from it).